tests/suites/test_suite_psa_crypto_pake.data

```diff
@@ -99,7 +99,7 @@ depends_on:PSA_WANT_KEY_TYPE_ECC_KEY_PAIR:PSA_WANT_ECC_SECP_R1_256:PSA_WANT_ALG_
 ecjpake_rounds:PSA_ALG_JPAKE:PSA_PAKE_PRIMITIVE(PSA_PAKE_PRIMITIVE_TYPE_ECC, PSA_ECC_FAMILY_SECP_R1, 256):PSA_ALG_SHA_256:PSA_ALG_TLS12_PSK_TO_MS(PSA_ALG_SHA_256):"abcdef":0:0:ERR_NONE
 
 PSA PAKE: check rounds w/o forced errors, TLS12_PRF
-depends_on:PSA_WANT_KEY_TYPE_ECC_KEY_PAIR:PSA_WANT_ECC_SECP_R1_256:PSA_WANT_ALG_SHA_256:PSA_WANT_ALG_TLS12_PSK_TO_MS
+depends_on:PSA_WANT_KEY_TYPE_ECC_KEY_PAIR:PSA_WANT_ECC_SECP_R1_256:PSA_WANT_ALG_SHA_256:A_WANT_ALG_TLS12_PRF
```

Comment:
Missing 2 letters? `A_WANT_` -> `PSA_WANT_`

Reply:
Ops, that's embarrassing, sorry...
I thought: "well it's only a single line to be changed, there's no need to test; what could possibly go wrong?" :/

Reply:
Yeah, I know the feeling :)

Actually just running the test suite wouldn't have caught it without taking a detailed look at the result: the test would have been skipped. I don't think it would have been caught by the CI either. We do have plans for this to be caught in the future, though. https://github.com/Mbed-TLS/mbedtls/issues/2691